pkg/tfvars/openstack/openstack.go

```diff
@@ -193,3 +215,147 @@ func getServiceCatalog(cloud string) (*tokens.ServiceCatalog, error) {
 
 	return serviceCatalog, nil
 }
+
+// createBootstrapSwiftContainer creates a container in swift with the bootstrap ignition config
```

Comment:
try moving these out to a separate file.

Reply:
Will do, its getting ugly in there